[IMP] pos: invoice issuance via qr code on receipts (#3411)
odoo/documentation: content/applications/sales/point_of_sale/shop/invoice.rst

.. note::
   - **Invoiced orders** can be identified by the :guilabel:`Invoiced` status in the
     :guilabel:`Status` column.
   - You can filter the list of orders to invoiced orders by clicking :guilabel:`Filters` and
     :guilabel:`Invoiced`.

QR codes to generate invoices
-----------------------------

> Customers can also request an invoice by scanning the **QR code** displayed on their receipt. Upon

Reviewer: ```suggestion
Customers can also request an invoice by scanning the **QR code** printed on their receipt. Upon
```

scanning, they must fill in a form with their billing information and click :guilabel:`Get my

> invoice`. By doing so, the invoice is generated and available to be downloaded. In Odoo backend, the

Reviewer: ```suggestion
invoice`. By doing so, the invoice is generated and available to be downloaded. In the backend of the POS app, the
```
Is there any a need to mention Odoo ? Do they ever leave Odoo for this feature ?

> invoice`. By doing so, the invoice is generated and available to be downloaded. In Odoo backend, the

Author: I like your suggestion better

.. note::
   - **Invoiced orders** can be identified by the :guilabel:`Invoiced` status in the
     :guilabel:`Status` column.
   - You can filter the list of orders to invoiced orders by clicking :guilabel:`Filters` and
     :guilabel:`Invoiced`.

QR codes to generate invoices
-----------------------------

Customers can also request an invoice by scanning the **QR code** printed on their receipt. Upon
scanning, they must fill in a form with their billing information and click :guilabel:`Get my
invoice`. On the one hand, doing so generates an invoice available for download. On the other hand,

> the order status goes from :guilabel:`Paid` or :guilabel:`Posted` to :guilabel:`Invoiced` in Odoo
> backend.

Reviewer 2: ```suggestion
the order status goes from :guilabel:`Paid` or :guilabel:`Posted` to :guilabel:`Invoiced` in the Odoo
backend.
```

.. image:: invoice/order-status.png
   :align: center
   :alt: order status change

> To allow this feature, you have to enable QR codes on receipts by going to :menuselection:`Point of

Reviewer 2: ```suggestion
To enable this feature, you have to enable QR codes on receipts by going to :menuselection:`Point of
```
Allowing a feature doesn't sound right. But I could be wrong :)
